SchoolDay.cpp

```diff
@@ -0,0 +1,51 @@
+#include "SchoolDay.h"
+
+SchoolDay::SchoolDay()
+{ 
+    dayOfTheWeek = "Monday";
+    length = 6.f;
+    runningLate = false;
+    std::cout << "A schoolday was created" << std::endl;
+
+}
```

Comment:
Always try to initialize member variables in the following order:
- in class
- in the constructor's member initializer list
- in the constructor body

the higher up that list, the better.

- [ ] revise